### DISCUSSION TITLE
Dates for Temp values from zigbee sensor not correct ?

### DISCUSSION THREAD

**Discussion by jgg74** [jmcollin78/versatile_thermostat#678] [General]
I have sometimes security mode on because the date of last temp is late. But it is strange , If I read  Zigbee2mqtt logs I received every 5 minutes values from the sensors , but in DEV tools -state the date of the last values doesn't match ( one hour) . I don't understand why. I don't have the same issue with sensors from RFXcom. bizarre. Any Ideas ? 
.-----..
J'ai quelques fois le security mode car la date de la dernière valeurs est en retard. Mais si je regarde les logs Zigbee2mqtt je reçois bien les valeurs toutes les 5 minutes.  Mais dans Outils de DEV - Etats , la date de la dernière valeur est une heure. 
Je ne comprends pas pourquoi . Je n'ai pas le même problème avec les capteurs RFXcom. Une idée ?

**Comment by jmcollin78** [comment]
In Zigbee you have the lastSeen entity (maybe you will be enable it). If you have one, you can configure it into the Vtherm main attributes so that this will avoid some safety mode switchs

**Comment by jgg74** [comment]
Thanks a lot Jmcollin78 to show me the solution. 
I followed this doc : https://www.hacf.fr/zigbee2mqtt-availability/ 
I changed param in z2m  because I had this one : 
![image](https://github.com/user-attachments/assets/5c870e3b-776c-43d2-8a66-ac3b47ef6ffc)
and after a restart of z2m I received the lastseen value 
![image](https://github.com/user-attachments/assets/482657a7-9653-49e8-91aa-6f618d052deb)
After I changed config in Vtherm : 
![image](https://github.com/user-attachments/assets/717bb8d1-9296-4e2e-bf53-82d64cfa84d4)

Again thanks for your help .
